src/mnode/ticket-processor.cpp

```diff
@@ -1088,6 +1093,60 @@ bool CPastelTicketProcessor::ParseP2FMSTransaction(const CMutableTransaction& tx
     return true;
 }
 
+std::vector<std::string> CPastelTicketProcessor::ValidateOwnership(const std::string &_txid, const std::string &_pastelID)
```

Comment:
replace return vector to bool or std::tuple?

Reply:
I'm not using tuple because of further possible improvements, in case for example we need to list for example all the copies owned by pastelId. Because currently this function returns IF pastel ID is owning at least one copy from the NFT.
Bool also cannot be used (without bigger modifications) because it is feed into the RPC response.